docs/storage/prometheus.md

```diff
@@ -14,79 +14,79 @@ To monitor cAdvisor with Prometheus, simply configure one or more jobs in Promet
 
```

Comment:
I think that is worth to mention and explain `disable_metrics` parameter in main section `# Monitoring cAdvisor with Prometheus`, near information about `prometheus_endpoint` parametr.

Reply:
Is it correct now?